erdpy_network/proxy_network_provider.py

```diff
@@ -128,6 +135,12 @@ def get_definition_of_token_collection(self, collection: str) -> DefinitionOfTok
         definition = DefinitionOfTokenCollectionOnNetwork.from_response_of_get_token_properties(collection, properties, self.address_hrp)
 
         return definition
+    
+    def simulate_transaction(self, transaction: Dict[str, Any]) -> Any:
+        raise NotImplementedError()
+    
+    def simulate_transaction_cost(self, transaction: Dict[str, Any]) -> Any:
+        raise NotImplementedError()
```

Comment:
Indeed, we should not implement this. We can remove it, and not call it from erdpy anymore.

Reply:
Removed it from network-providers.